### DISCUSSION TITLE
When will minio generate .minio.sys/tmp-old folder

### DISCUSSION THREAD

**Discussion by garyyang85** [minio/minio#16144] [Q&A] [answered]
When will minio generate the ".minio.sys/tmp-old" folder? We have 3 minio file servers, only one has this folder.  I found there are millions of files in this folders, can I manually delete them? Thanks.

**Comment by klauspost** [comment, marked as answer]
On restart. Yes, you can delete it. 

Also recently fixed abandoned `tmp-old` folders: https://github.com/minio/minio/pull/15979

**Comment by youtehub** [comment]
Can I set the saving path of.minio.sys separately? Now it is generated directly. It is inconvenient to do so under the path deployed at startup, can I support it? Like./minio server /minio/rootfolder --dir-file /minio/sysconfig --console-address ":8001" --address ":9001" sys is automatically generated into the -- --dir-file /minio/sysconfig directory instead of the /minio/rootfolder in the startup path. For some special reason, when deployed now, The automatic generation in the /minio/rootfolder will cause some problems, making minio not work properly

**Reply by klauspost** [reply]
No. This is fixed.
